lib/Valetudo.js

```diff
@@ -48,6 +49,30 @@ const Valetudo = function() {
         vacuum: this.vacuum,
     });
 
+    if(this.configuration.get("telegram_token")) {
+        this.telegramClient = new Telegrambot({ 
+            vacuum: this.vacuum, 
+            events: this.events, 
+            configuration: this.configuration, 
+            clients: this.configuration.get("telegram_clients"), 
```

Comment:
Why this and following variables set here and not inside Telegrambot constructor?

Reply:
I need the variables vacuum, events, ... i have no idea how i can access this variables inside the telegramobject. I add the configuration later, so i don't need to transfer the variables like telegram_token. Is there a another way to access vaccum or events?

Reply:
That was comment about passing some specific configuration properties along with configuration object itself. Vacuum and events were quoted here because github shows some context before the commented line it seems.